dotCMS/src/main/java/com/dotcms/security/apps/SecretsStoreKeyStoreImpl.java

```diff
@@ -286,26 +285,21 @@ private Key key() {
     }
 
     @VisibleForTesting
-    protected String encrypt(final char[] val) {
-        return encrypt(new String(val));
-    }
-
-    @VisibleForTesting
-    protected String encrypt(final String val) {
-        return Sneaky.sneak(() -> Encryptor.encrypt(key(), val));
+    protected char[] encrypt(final char[] val) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider using varargs for methods or constructors which take an array the last parameter.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5329810)